src/main/java/domain/user/Users.java

```diff
@@ -0,0 +1,81 @@
+package domain.user;
+
+import static domain.game.Result.LOSE;
+import static domain.game.Result.WIN;
+
+import domain.TotalDeck;
+import domain.card.Card;
+import domain.game.Result;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Users {
+    private static final int DEALER_INDEX = 0;
+    private static final int FIRST_PLAYER_INDEX = 1;
+    private final List<User> users;
+
+    public Users(List<Player> players) {
+        validateDuplicatedName(players);
+        users = new ArrayList<>();
+        users.add(DEALER_INDEX, new Dealer());
+        users.addAll(players);
+    }
+
+    public void setStartCards(TotalDeck totalDeck) {
+        for (User user : users) {
+            user.addCard(totalDeck.getNewCard());
+            user.addCard(totalDeck.getNewCard());
+        }
+    }
+
+    public boolean isDealerCardAddable() {
+        return getDealer().isAddable();
+    }
+
+    public void addDealerCard(Card card) {
+        getDealer().addCard(card);
+    }
+
+    public Result generatePlayerResult(Player player) {
+        if (player.busted()) {
+            return LOSE;
+        }
+        if (getDealer().busted()) {
+            return WIN;
+        }
+        return Result.compare(player.sumUserDeck(), getDealer().sumUserDeck());
+    }
+
+    public List<User> getUsers() {
+        return Collections.unmodifiableList(users);
+    }
+
+    public List<Player> getPlayers() {
+        List<User> players = users.subList(FIRST_PLAYER_INDEX, users.size());
+        return players.stream()
+                .map(user -> (Player) user)
+                .collect(Collectors.toList());
+    }
+
+    private User getDealer() {
+        return users.get(DEALER_INDEX);
+    }
+
+    private void validateDuplicatedName(List<Player> players) {
+        List<Player> distinctPlayers = new ArrayList<>();
+        players.forEach(player -> {
+            throwExceptionIfContains(distinctPlayers, player);
+            distinctPlayers.add(player);
+        });
+    }
+
+    private static void throwExceptionIfContains(List<Player> distinctPlayers, Player player) {
+        if (distinctPlayers.contains(player)) {
+            throw new IllegalArgumentException(
+                    "중복된 이름은 입력할 수 없습니다: %s".formatted(player.getNameValue())
+            );
+        }
+    }
```

Comment:
```suggestion
    private void throwExceptionIfContains(List<Player> distinctPlayers, Player player) {
        if (distinctPlayers.contains(player)) {
            throw new IllegalArgumentException("중복된 이름은 입력할 수 없습니다: %s".formatted(player.getNameValue()));
        }
    }
```
한칸을 띄우지 않아도 line이 벗어나지 않는데 띄운 이유가 있을까요? 보통은 이렇게 괄호를 통으로 내리진 않아요.

Reply:
한줄을 띄우지 않아도 라인 컨벤션을 벗어나진 않지만, 에러 메세지가 긴 상황에서 줄을 나누는 것이 더 보기 편하게 느껴졌습니다.

하지만, 저 괄호 하나만 내려진 부분은 불필요한 개행이라는 생각이 드네요.

수정했습니다!